aiohomekit/controller/ip/connection.py

```diff
@@ -370,7 +389,9 @@ async def put_json(self, target, body):
 
         return parsed
 
-    async def post(self, target, body, content_type=HttpContentTypes.TLV):
+    async def post(
+        self, target: str, body: Any, content_type=HttpContentTypes.TLV
```

Comment:
And here

Reply:
38cdf65d6aaa80edc21a02d9ac432a9b1daae5a7

Reply:
I guess this one needs to not be none

Reply:
073049de3d622ef50d1e91bcecfb5cc6f01735fc